docs/contrib/first_time.md

```diff
@@ -0,0 +1,244 @@
+---
+title: How to become an Open Source Contributor
+---
+
+Welcome, potential new Open Source contributor! This is a guide to show you
```

Comment:
In general this document needs to be adapted to the new branching model (only `stable`/`develop` branches)